code/game/machinery/medical_kiosk.dm

```diff
@@ -237,7 +237,7 @@
 			if(R.overdosed)
 				overdose_list += list(list("name" = R.name))
 	var/obj/item/organ/stomach/belly = altPatient.getorganslot(ORGAN_SLOT_STOMACH)
-	if(belly?.reagents.reagent_list.len) //include the stomach contents if it exists
+	if(belly?.reagents?.reagent_list.len) //include the stomach contents if it exists
```

Comment:
Ditto here, the ?. is only there because belly getorganslot can return null, reagents ought to always exist on stomachs, unless something is very wrong